redaxo/src/core/lib/console/db/set_connection.php

```diff
@@ -29,7 +29,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $io = $this->getStyle($input, $output);
 
         $configFile = rex_path::coreData('config.yml');
-        $config = rex_file::getConfig($configFile, ['db' => [1 => []]]);
+        $config = rex_file::getConfig($configFile);
+
+        $config = $config + ['db' => [1 => ['host' => '', 'login' => '', 'password' => '', 'database' => '']]];
```

Comment:
```suggestion
        $config['db'][1] = array_merge(['host' => '', 'login' => '', 'password' => '', 'database' => ''], $config['db'][1] ?? []);
```

Ich denke, so sollte es funktionieren.

Reply:
funktioniert auch nicht :-)

Reply:
Jupp, ich kann lokal mal testen, und mir eine funktioniere Variante überlegen. Müssen hier ja sowieso auch noch auf den psalm fix warten.